clients/agentic-sandbox-client-python/README.md

```diff
@@ -0,0 +1,98 @@
+
+```bash
+pip install -e .
+```
+
+**Example:**
+
+```python
+from agentic_sandbox import Sandbox
+
+with Sandbox(template_name="sandbox-python-template", namespace="default") as sandbox:
+    result = sandbox.run("echo 'Hello, World!'")
+    print(result.stdout)
+```
+
+## How It Works
+
+The `Sandbox` client automates the entire lifecycle of a temporary sandbox environment:
+
+1.  **Initialization (`with Sandbox(...)`):** The client is initialized with the name of the `SandboxTemplate` you want to use and the namespace where the resources should be created:
+
+-   **`template_name` (str):** The name of the `SandboxTemplate` resource to use for creating the sandbox.
+-   **`namespace` (str, optional):** The Kubernetes namespace to create the `SandboxClaim` in. Defaults to "default". 
+-   When you create a `Sandbox` instance within a `with` block, it initiates the process of creating a sandbox.
+2.  **Claim Creation:** It creates a `SandboxClaim` Kubernetes resource. This claim tells the agent-sandbox controller to provision a new sandbox using a predefined `SandboxTemplate`.
+3.  **Waiting for Readiness:** The client watches the Kubernetes API for the corresponding `Sandbox` resource to be created and become "Ready". This indicates that the pod is running and the server inside is active.
+4.  **Port Forwarding:** Once the sandbox pod is ready, the client automatically starts a `kubectl port-forward` process in the background. This creates a secure tunnel from your local machine to the sandbox pod, allowing you to communicate with the server running inside.
+5.  **Interaction:** The `Sandbox` object provides three main methods to interact with the running sandbox:
+    *   `sandbox.run(command)`: Executes a shell command inside the sandbox.
+    *   `sandbox.write(path, content)`: Uploads a file to the sandbox.
+    *   `sandbox.read(path)`: Downloads a file from the sandbox.
+6.  **Cleanup (`__exit__`):** When the `with` block is exited (either normally or due to an error), the client automatically cleans up all resources. It terminates the `kubectl port-forward` process and deletes the `SandboxClaim`, which in turn causes the controller to delete the `Sandbox` pod.
+
+
+## How to Test the Client
+
+A test script, `test_client.py`, is included to verify the client's functionality.
+You should see output indicating that the tests for command execution and file operations have passed.
+
+## Packaging and Installation
+
+This client is configured as a standard Python package using `pyproject.toml`.
+
+### Prerequisites
+
+-   Python 3.7+
+-   `pip`
+-   `build` (install with `pip install build`)
+
+### Building the Package
+
+To build the package from the source, navigate to the `agentic-sandbox-client-python` directory and run the following command:
+
+```bash
+python -m build
+```
+
+This will create a `dist` directory containing the packaged distributables: a `.whl` (wheel) file and a `.tar.gz` (source archive).
+
+### Installing the Package
```

Comment:
Shouldn't this be part of the "### Installation" above?

You can have a simple installation for end users, and then another option to install from source for contributors.